gcore/gdaldataset.cpp

```diff
@@ -8437,6 +8437,58 @@ void GDALDatasetClearStatistics(GDALDatasetH hDS)
     GDALDataset::FromHandle(hDS)->ClearStatistics();
 }
 
+
+/************************************************************************/
+/*                        GetFieldDomains()                              */
+/************************************************************************/
+
+/** Returns a list of the names of all field domains stored in the dataset.
+ *
+ * @param papszOptions Driver specific options determining how attributes
+ * should be retrieved. Pass nullptr for default behavior.
+ *
+ * @return list of field domain names
+ * @since GDAL 3.5
+ */
+std::vector<std::string> GDALDataset::GetFieldDomainNames(CSLConstList) const{
```

Comment:
to make Doxygen happy
```suggestion
std::vector<std::string> GDALDataset::GetFieldDomainNames(CPL_UNUSED CSLConstList papszOptions) const{
```